config-linux.md

```diff
@@ -186,7 +186,7 @@ You can configure a container's cgroups via the `resources` field of the Linux c
 Do not specify `resources` unless limits have to be updated.
 For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
 
-A runtime MUST at least use the minimum set of cgroup controllers required to fulfill the `resources` settings.
+A runtime MUST at least use the minimum set of cgroup controllers necessary to fulfill the `resources` settings.
 However, a runtime MAY attach the container process to additional cgroup controllers supported by the system.
```

Comment:
Do we even want to keep the line you're bumping?  It sounds like any runtime which violated that constraint would necessarily violate some more specific constraint on a `resources` setting.  Maybe replace this paragraph with:

> Runtimes MAY attach the container process to additional cgroup controllers beyond those required to fulfill the `resources` settings.